mscore/editstyle.cpp

```diff
@@ -743,24 +744,36 @@ void EditStyle::on_comboFBFont_currentIndexChanged(int index)
 
 void EditStyle::on_buttonTogglePagelist_clicked()
       {
-      if (pageList->isVisible()) {
-            pageList->setVisible(false);
-            if (!isTooWide) {
```

Comment:
No more repeated literal values of 15 and 5.  Just a mechanism for stashing the current size and restoring it later.  This also allows a resized dialog box to be properly restored, instead of always reverting to the default size when this function runs.